### DISCUSSION TITLE
on coupon page, what's meant by "parent only" and "children only" when add a condition or how it works

### DISCUSSION THREAD

**Discussion by o414** [bagisto/bagisto#7387] [General]
on the coupon page, what's meant by "parent only" and "children only" when adding a condition, or how it works?

**Comment by Sanjaybhattwebkul** [comment]
Hi there,
Please **follow** [this thread](https://github.com/bagisto/bagisto/issues/7389).

**Comment by Amitk-webkul** [comment]
Hi @o414, I believe you've found your solution. If you have any more questions in the future, don't hesitate to reach out. Just go ahead and close this discussion for now.
